events/wc_dark_cult_events.txt

```diff
@@ -1114,6 +1159,231 @@ narrative_event = {
 				character_event = { id = MNM.10030 }
 			}
 			narrative_event = { id = WCDAC.5050 }
+			character_event = { id = WCDAC.5041 days = 7 }
+			character_event = { id = WCDAC.5045 years = 5 }
+		}
+	}
+}
+character_event = {
+	id = WCDAC.5041
```

Comment:
So, ooof, yeah, Garona fell onto her own dagger.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/79171775-02ca4200-7e04-11ea-9456-e1fc2a001529.png)

</details>

Reply:
fixed